tests/installation/installer_desktopselection.pm

```diff
@@ -36,6 +42,12 @@ sub run() {
     send_key 'spc';                                                            # Select the desktop
 
     assert_screen "$d-selected";
+    if (check_var('VERSION', 'Tumbleweed')) {
```

Comment:
so should I create some variable half year before products exists ?

`check_var('VERSION', 'Tumbleweed') && check_var('VERSION', '15')`